### DISCUSSION TITLE
Excluding auto-imports for utils dir

### DISCUSSION THREAD

**Discussion by maou-shonen** [nitrojs/nitro#1356] [Q&A]
I want to exclude utils dir from auto-imports,
It should be possible with the `imports.exclude` option,
but I'm not sure how to match it.

I have already tried.
```ts
  imports: {
    exclude: [
      /\/utils\//,
      'utils',
      'utils/*',
      'utils/**',
    ],
  },
```

**Comment by ngregrichardson** [comment]
Wondering if anyone has an answer to this...although this works with my IDE, I'm not able to get it to work properly when my tests (Vitest) run. Any function coming from utils is just not defined.

**Reply by ngregrichardson** [reply]
`autoImports: false` removes the built-in functions from the global scope, but not the utils folder

**Comment by rafael-lua** [comment]
Same question. I'm disabling auto-imports on a nuxt project, but the nitro side still keeps auto-importing. At least, I wish it to stop auto-importing from utils folder. My nuxt config has:

```
nitro: {
    imports: {
      autoImport: false,
    }
},
imports: {
    autoImport: false,
}
```

**Comment by XStarlink** [comment]
Hello @pi0, really sorry to ping you like this,

Like all of us here I'm also looking for a way to disable autoImport for the server/utils folder (because I'm having name collisions on a big Nuxt 3 project), I've tried lots of ways but I haven't found how to do it.

Can you please guide us?
